cfme/tests/intelligence/reports/test_metering_report.py

```diff
@@ -304,3 +312,16 @@ def test_validate_disk_usage(resource_usage, metering_report):
             assert estimated_disk_usage - DEVIATION <= float(usage) \
                 <= estimated_disk_usage + DEVIATION, 'Estimated cost and report cost do not match'
             break
+
+
+def test_validate_storage_usage(resource_usage, metering_report):
+    """Test to validate storage usage."""
+    for groups in metering_report:
+        if groups["Storage Used"]:
+            estimated_storage_usage = resource_usage['storage_used']
+            usage_from_report = groups["Storage Used"]
+            usage = re.sub(r'[MB, GB,]', r'', usage_from_report)
```

Comment:
REQUIRED: What is the purpose of this regex? To strip 'MB' or 'GB' from the string? If that's the case, it should be ``r'(MB|GB)'`` (or ``r' ?(MB|GB)'`` if you there can be a space that you want to strip as well).
What you wrote is equivalent to ``r'[GMB, ]'`` and will strip any G, M, B, comma or space character. If it works in this case it's just coincidental.
It should be fixed in other places as well.

Reply:
@mkoura , Fixed in all the places